ports/beemiscurseofgod/gameinfo.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<gameList>
+  <game>
+    <path>./Beemis - The Curse of God.sh</path>
+    <name>Beemis: The Curse of God</name>
+    <desc>Become a real life bee and also fight god himself. Take on many bosses and wacky levels as a young bee whose life has been drastically changed by a curse maniacally concocted by God.</desc>
+    <releasedate>20180611T000000</releasedate>
+    <developer>blebuel</developer>
+    <publisher>Self-Published</publisher>
+    <genre>Action</genre>
+    <image>./beemiscurseofgod/screenshot.png</image>
```

Comment:
Should be ./beemisthecurseofgod/screenshot.png

Reply:
Thanks, done.